wsmaster/che-core-api-devfile/src/test/resources/devfile.yaml

```diff
@@ -28,6 +28,12 @@ tools:
   - name: jdt.ls
     type: chePlugin
     id: eclipse/theia-jdtls:0.0.3
+  - name: k8s-recipe
```

Comment:
We are not going to support more than one tool that is not `cheEditor` nor `chePlugin`. It would be better not to have such example even in test sources

Reply:
ok